src/util/api.js

```diff
@@ -51,26 +51,34 @@ const getNibrsRequests = params => {
   return slices.map(s => getNibrs({ ...s, crime, place }))
 }
 
-const getSummary = params => {
-  const { place } = params
-  const endpoint = place === nationalKey
+const getSummaryEndpoint = ({ place, placeType }) => {
+  if (placeType === 'agency') {
+    const state = place.slice(0, 2)
+    return `${API}/agencies/count/states/offenses/${state}/${place}`
```

Comment:
I realize this is the endpoint, but I've [proposed that the URL](https://github.com/18F/crime-data-api/issues/492) be changed to something a bit easier to work with such as `/agencies/count/offenses/:ORI`. If we can infer the state from the ORI, I think the API should just do that for you so the user needs to know less.